glass/shapes.py

```diff
@@ -87,12 +91,20 @@ def triaxial_axis_ratio(zeta, xi, size=None, *, rng=None):  # type: ignore[no-un
     C = 1 + z2m1 * cos2_phi  # noqa: N806
 
     # eq. (12)
-    return np.sqrt(
+    return np.sqrt(  # type: ignore[no-any-return]
         (A + C - np.sqrt((A - C) ** 2 + B2)) / (A + C + np.sqrt((A - C) ** 2 + B2)),
     )
```

Comment:
👀 Is that a stray comma at the end of the line that causes mypy to complain?

Reply:
No. The `no-any-return` I believe is because the result of `np.sqrt` is meant to be `npt.NDArray[np.float64]` but the actual result isn't guaranteed to be one. So we could use `.astype`. This is why I haven't fixed that one.